ui-components/src/actions/ExpandableContent.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react"
+import { t } from "@bloom-housing/ui-components"
+
+type ExpandableContentProps = {
+  children: React.ReactChild
+}
+
+const ExpandableContent = ({ children }: ExpandableContentProps) => {
```

Comment:
I realize this is sort of based on code I'd added in the previous preferences pages￼…but looking at this fresh I'm wondering if we can refactor to use a simple details/summary markup in the HTML directly. Prior art in the WhatToExpect component. Thoughts?

Reply:
Hmm, we can - but here we have two different labels: read less / read more.
We can create a listener to watch this state and show the proper label, but I'm not sure which one is simpler in this case :D 

Also - this solution is open e.g for animation in the future 💅

Reply:
Good point. Maybe we can take a second look at this later on and see if it's something to standardize on for all the read more/less instances around the codebase.